Sources/SwiftParser/Parameters.swift

```diff
@@ -156,7 +156,18 @@ extension Parser {
       defaultValue = nil
     }
 
-    let trailingComma = self.consume(if: .comma)
+    var trailingComma: Token?
+    if self.at(.comma) {
+      trailingComma = self.consume(if: .comma)
+    } else if !self.at(.rightParen) {
+      let canParseIdentifier: Bool = withLookahead {
+        $0.canParseTypeIdentifier(allowKeyword: false) || $0.canParseCustomAttribute()
```

Comment:
```swift
    assertParse(
      "func f(_: any borrowing 1️⃣~Copyable) {}",
      diagnostics: [
        DiagnosticSpec(
          message: "unexpected code '~Copyable' in parameter clause"
        )
      ]
    )
```

This test is failing @ahoppen. 

```
failed - Expected 1 diagnostics but received 2:
1:24: expected ',' in parameter
1:25: expected identifier and ':' in parameter
```

Not sure how to handle this?

Reply:
`$0.canParseCustomAttribute()` return true for `~Copyable`

Reply:
Hmm, let’s just remove `canParseCustomAttribute` then. Attributes on parameters aren’t very common anyway.

Reply:
@ahoppen I added an `self.at(.atSign)` that fixes the issue. 
I self only know `@` attributes for names. Is there others?

Reply:
Changed it to consuming it.